src/App.js

```diff
@@ -1,15 +1,31 @@
 import './App.scss';
-
 import React, { useEffect } from 'react';
-
 import { RouteList } from './Routes';
+import { useChrome } from '@redhat-cloud-services/frontend-components/useChrome';
+import { getRegistry } from '@redhat-cloud-services/frontend-components-utilities/Registry';
+import { notificationsReducer } from '@redhat-cloud-services/frontend-components-notifications/redux';
+import NotificationsPortal from '@redhat-cloud-services/frontend-components-notifications/NotificationPortal';
 
 const App = (props) => {
+  const chrome = useChrome();
+
   useEffect(() => {
     insights.chrome.hideGlobalFilter?.();
   }, []);
 
-  return <RouteList childProps={props} />;
+  useEffect(() => {
+    if (chrome) {
+      const registry = getRegistry();
+      registry.register({ notification: notificationsReducer });
+    }
+  }, [chrome]);
```

Comment:
Why is this required? The store `init` function should already have the notificationsReducer initialised, does it not?